src/GovUk.Education.ExploreEducationStatistics.Data.Services/ReleaseService.cs

```diff
@@ -71,6 +72,18 @@ private async Task<List<SubjectViewModel>> GetSubjects(Guid releaseVersionId, Li
                 .Where(rs => rs.ReleaseVersionId == releaseVersionId && subjectsToInclude.Contains(rs.SubjectId))
                 .ToListAsync();
 
+            if (!ComparerUtils.SequencesAreEqualIgnoringOrder(
+                    releaseSubjects.Select(rs => rs.SubjectId).ToList(),
+                    subjectsToInclude
+                    ))
```

Comment:
Not one for now but I'm wondering if we should eventually get rid of that util method in favour of something like this where it's clearer there's still ordering going on, and it's shorter!

`subjectsToInclude.Order().SequenceEqual(releaseSubjects.Select(rs => rs.SubjectId).Order())`

Reply:
I'm not really fussed about it - I think either way it's clear what it does at a glance and there isn't any ambiguity which means you need to look inside the method. If I were starting from scratch, I'd prefer the new way you're proposing.

So you're right I do think this is too minor to be spending any time on :)